mlflow/store/db/utils.py

```diff
@@ -197,6 +211,25 @@ def create_sqlalchemy_engine(db_uri):
         pool_kwargs["pool_recycle"] = pool_recycle
     if echo:
         pool_kwargs["echo"] = echo
+    if poolclass:
+        pool_class_map = {
+            "AssertionPool": AssertionPool,
+            "AsyncAdaptedQueuePool": AsyncAdaptedQueuePool,
+            "FallbackAsyncAdaptedQueuePool": FallbackAsyncAdaptedQueuePool,
+            "NullPool": NullPool,
+            "QueuePool": QueuePool,
+            "SingletonThreadPool": SingletonThreadPool,
+            "StaticPool": StaticPool,
+        }
+        if poolclass not in pool_class_map:
+            list_str = " ".join(pool_class_map.keys())
+            err_str = (
+                "Invalid poolclass parameter. Provide set environment variable "
+                "poolclass to empty or one of the following values: " + list_str
+            )
+            _logger.warning(err_str)
```

Comment:
Do we need this line?

Reply:
My understanding is that the "raise" gives user visibility. And the logger gives us(databricks engineers) visibility about it's happening. 
(Not sure if the loggers are the same as loggers in scala)

Reply:
Does the "raise" also give us visibility about it's happening?